lib/archethic/account/mem_tables/token_ledger.ex

```diff
@@ -176,6 +227,9 @@ defmodule Archethic.Account.MemTables.TokenLedger do
       [] ->
         DB.get_inputs(:token, address)
 
+      [{^address, true}] ->
+        DB.get_inputs(:token, address)
```

Comment:
You should remap spent? to false

Reply:
[{^address, true}] ->
I am not sure if you are asking about ths line, this means the inputs reached threshold for that address and were moved to db, were always be spent

Reply:
As spent is to be removed in an another pr

Reply:
I don't think, just make it as spent, keeping spent? on disk save I/O when the input will be really spent.